app/src/pages/inside/milestonesPage/emptyMilestones/emptyMilestones.tsx

```
@@ -0,0 +1,61 @@
+import { Button } from '@reportportal/ui-kit';
+import { referenceDictionary } from 'common/utils';
+import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+import { ExternalLink } from 'pages/inside/common/externalLink';
+import { messages } from './messages';
+import styles from './emptyMilestones.scss';
+
+const cx = classNames.bind(styles);
+
+type EmptyMilestonesProps = {
+  // add required props types
+};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)